src/vision/scripts/imgPublisher.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+# license removed for brevity
+
+# import the necessary packages
+from __future__ import print_function
+from pyimagesearch.basicmotiondetector import BasicMotionDetector
+from pyimagesearch.panorama import Stitcher
+from imutils.video import VideoStream
+import numpy as np
+import datetime
+import imutils
+import time
+import cv2
+
+import rospy
+from sensor_msgs.msg import Image
+from cv_bridge import CvBridge, CvBridgeError
+
+# initialize the video streams and allow them to warmup
+print("[INFO] starting cameras...")
+leftStream = VideoStream(src=0).start()
+rightStream = VideoStream(src=1).start()
+time.sleep(1.0)
+
+# initialize the image stitcher, motion detector, and total
+# number of frames read
+stitcher = Stitcher()
+motion = BasicMotionDetector(minArea=500)
+total = 0
+
+ImgStitcher = rospy.Publisher('ImgStitcher', Image, queue_size=10) 
+rospy.init_node('ImgPublisher', anonymous=True)
```

Comment:
We don't want anonymous=True here, as this will just spawn this node with a different name if there is already an instance of it running. If we create another instance of this node, we want the first one to automatically be killed, because we don't want two programs trying to access USB devices at the same time. Please go through the rest of this PR and remove any other instances of it as well.